File: Python/1015.py
import math

lines1 = input().split(" ")
lines2 = input().split(" ")

y1 = float(lines1.pop())
x1 = float(lines1.pop())

y2 = float(lines2.pop())
x2 = float(lines2.pop())

dist = math.sqrt(pow(x2 - x1, 2) + pow(y2 - y1, 2))

print(f"{dist:.4f}")

File: JavaScript/1015.js
let input = require('fs').readFileSync('./dev/stdin/file.txt', 'utf8');
let lines = input.split('\n');
let coordenadas1 = lines[0].split(' ');
let coordenadas2 = lines[1].split(' ');

let x1 = parseFloat(coordenadas1.shift());
let y1 = parseFloat(coordenadas1.shift());
let x2 = parseFloat(coordenadas2.shift());
let y2 = parseFloat(coordenadas2.shift());

let distancia = Math.sqrt(Math.pow(x2 - x1, 2) + Math.pow(y2 - y1, 2));

console.log(distancia.toFixed(4));